src/inmanta/data/model.py

```diff
@@ -287,11 +285,47 @@ def level_from_enum_attribute(cls, v: object) -> object:
             try:
                 return const.LogLevel[v]
             except KeyError:
-                raise ValueError(f"Invalid enum value {v}. Valid values: { ','.join([x.name for x in const.LogLevel]) }")
+                raise ValueError(f"Invalid enum value {v}. Valid values: {','.join([x.name for x in const.LogLevel])}")
         return v
 
     level: const.LogLevel
     msg: str
     args: List[Optional[ArgumentTypes]] = []
     kwargs: Dict[str, Optional[ArgumentTypes]] = {}
     timestamp: datetime.datetime
+
+
+class ResourceIdDetails(BaseModel):
+    resource_type: ResourceType
+    agent: str
+    attribute: str
+    value: str
+
+
+class ResourceListElement(BaseModel):
```

Comment:
I find it weird to have `ListElement` as part of the class name. It is plausible that we will add an API endpoint in the future that return this type, while it's not part of a list, no?